libcloud/container/drivers/rancher.py

```diff
@@ -41,7 +41,7 @@ class RancherResponse(JsonResponse):
     def parse_error(self):
         if self.status == 401:
             raise InvalidCredsError('Invalid credentials')
-        return self.body
+        return super().parse_error()
```

Comment:
You can just take this method out altogether and let it fall through to the base class if you're not going to do anything Rancher-specific with the response.